src/Routes.js

```diff
@@ -134,7 +134,16 @@ router.get(`/${apiVersion}/${enginesEndpoint}`, async (ctx) => {
  *         description: Task information in verbatim format as returned by the Docker Engine Remote API.
  *       kubernetes:
  *         type: object
- *         description: Pod information in verbatim format as returned by the Kubernetes API.
+ *         properties:
+ *           pod:
+ *             type: object
+ *             description: Pod information in verbatim format as returned by the Kubernetes API.
+ *           replicaSet:
```

Comment:
How do we document that this key might not exist? Or, will it always exist but with possible value `undefined`? Should we clarify this?

Reply:
I think the key will not exist, but I will verify and add that info in the docummentation